XML/TestCases/PerformanceTests.xml

```diff
@@ -770,4 +770,24 @@
       <OverrideVMSize>Standard_D3_v2</OverrideVMSize>
     </SetupConfig>
   </test>
-</TestCases>
\ No newline at end of file
+  <test>
+    <testName>PERF-XDP-TCP-LATENCY</testName>
+    <testScript>PERF-XDP-TCP-LATENCY.ps1</testScript>
+    <files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\XDPDumpSetup.sh,.\Testscripts\Linux\perf_lagscope.sh</files>
+    <TestParameters>
+      <param>pingIteration=LAGSCOPE_PING_ITERATIONS</param>
```

Comment:
set it to 1000000 directly?

Reply:
I did it originally, but later with reviewer's comments found out that it is better to keep a replaceable parameter to control all values with one parameter.

Reply:
fixed it in the latest commit.